tdrs-frontend/.pa11yci.json

```diff
@@ -0,0 +1,12 @@
+{
+  "standard": "WCAG2AAA",
```

Comment:
I like seeing the goal of AAA for WCAG2. I'll be curious about seeing whether the AAA setting creates any issues with passing CI tests as the work progresses.